README.md

```diff
@@ -30,15 +30,17 @@ Changelogs are maintained in [CHANGELOG.md](CHANGELOG.md)
 ### 🤗 Local Installation & Contributing
 
 - Fork [abelljs/abell](https://github.com/abelljs/abell)
-- `git clone https://github.com/:github-username/abell`
-- `cd abell`
+- `git clone https://github.com/:github-username/abell` ⏬
```

Comment:
Usually, people double click to select a line to copy, if we add this icon at the end and someone selects the line, they will get `git clone https://github.com/:github-username/abell arrow_double_down` in the clipboard. So maybe we can remove that icon from there.

Also, I realized even in the lines below I've added text that is being copied on selecting a line.

One solution I can think of is to have all these commands in a single block and write comments in `#`

```sh
git clone 
npm install
npm link # This text will be copied but terminal understands that this is a comment and ignore this text in the command
```

Something like this ^^. Will you be interested in making these changes?

Reply:
Sure